### ISSUE TITLE
List of new issues to be opened

### ISSUE THREAD

**Issue by santoshphilip** [pyenergyplus/witheppy#3]
While resolving issue #1 an number of new tasks opened up. I am listing them here so that new issues can be opened. Many of these are one time tasks arising from the intial setup of an open source repository where many people are expected to contribute

- Use NumPy Style Python Docstrings. The docstring in the code looks like <https://pyclearsky.readthedocs.io/en/latest/_modules/pyclearsky/clearskyrad.html#ETradiation>. Corresponding html generated by sphinx looks like <https://pyclearsky.readthedocs.io/en/latest/pyclearsky.html#pyclearsky.clearskyrad.ETradiation>. The numpydoc docstring fuide is here: <https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard>
    - issue opened
    - issue fixed
- Set up tests so that the idd is read only once during pytests. Otherwise the test take a long time as it takes a long time to read the idd file.  Eppy does this in it's tests. Improve the eppy method, since it has issues
    - issue opened
- have extfields.extensiblefields2list() and extfields.extensiblefields2list .list2extensiblefields() return nested lists
    - issue completed and closed
- write a function to return first extensible field names - first_extfieldnames(idfobject). Looks like this should be usefull
- set up continious testing
    - issue opened
    - issue is on hold. appveyor is failing on this account. It works on santoshphilip account
- documentation for developers on the process/protocol. See <https://github.com/pyenergyplus/py90dot1/blob/master/CONTRIBUTING.md> as a sample
    - issue completed and closed
- develop user documentation and examples in the /docs folder
    - issue opened
- deploy docs on <https://readthedocs.io/>
    - issue completed and closed
- deploy on <>PyPi <https://pypi.org>
    - issue completed and closed
- Add content to the README.rst
    - issue completed and closed
